### DISCUSSION TITLE
Documents effacés sans raison

### DISCUSSION THREAD

**Discussion by artesburger** [dadoonet/fscrawler#1509] [Q&A]
Bonjour,

J'ai lancé fscrawler et je l'ai laissé tourner pendant plus d'une heure.
J'étais arriver à plus de 20000 docs indexé, et d'un coup je suis tombé à 3000 fichiers. 
Quand je regarde sur kibana, j'ai 190000 fichiers deleted.
![image](https://user-images.githubusercontent.com/92673359/192516561-8ffef311-4181-45c9-8344-7455d2ea298d.png)

Savez vous d'ou ça vient ?

**Comment by dadoonet** [comment]
> Cela peut il venir du fait que j'ai changé de réseaux pendant ce temps, et donc fscrawler ne pouvant plus ce connecter au chemin indiquer a effacé des documents ?

Oui. Si `remove_deleted` est à `true` (valeur par défaut).

Regarde https://fscrawler.readthedocs.io/en/latest/admin/fs/local-fs.html#ignore-deleted-files
